tests/test_web/test_api_v1.py

```diff
@@ -84,10 +84,12 @@ def test_get_build(app, auth_env, client, db):
 
 
 def test_get_builds(app, auth_env, client, db):
-    total_requests = 50
+    total_create_requests = 5
```

Comment:
It would be nice to test the filters in a separate function. But I can look the other way on this